### ISSUE TITLE
Add Support for DCS Open Beta (disabled devices)

### ISSUE THREAD

**Issue by ereidelbach** [Rexeh/joystick-diagrams#40]
The current version (1.2) does not appear to work with the Open Beta branch of DCS.  When I run the program and direct it to my Saved Games folder (Users/XXX/Saved Games/DCS.openbeta) the program throws the following error:

Error: [WinError 267] The directory name is invalid: 'C:/Users/XXX/Saved Games/DCS.openbeta\\Config\\Input\\disabled.lua'

**Comment by Sturmovik-ice** [comment]
I got it to work in Open Beta by deleting the 'disabled.lua' file in the Input folder.

**Comment by ereidelbach** [comment]
Thanks for sharing that as a fix.  Works on my end as well.  For now I can at least move that file out, generate my diagrams, and then move it back.

**Comment by Rexeh** [comment]
Cheers for this, not expecting LUA in here so not sure why you have it. Do you have reproduce steps on how you get a disabled.lua in this folder?

**Comment by ereidelbach** [comment]
Hey Rexeh, I sure do...

If you go into the in-game options and select the Controls tab, ED now gave us the option to disable certain controllers.  In my case, I disabled my XBox Game Pad so that it would no longer show up as a possible device to which to bind controls.

Screenshot:
![disable_control](https://user-images.githubusercontent.com/30529545/103173535-b60b8780-4829-11eb-8d4b-26a8f316b77b.png)

If this option is used, it appears DCS generates a "disabled.lua" file and places it in the Config/Input folder.  

For me, it generated a file which contained the following:
`local disabled = {
	["devices"] = {
		["2 axis 8 button gamepad {7A5A3D00-8702-11ea-8001-444553540000}"] = true,
	},
	["pnp"] = false,
}
return disabled`

Hope this helps!

-Stewie

**Comment by Sturmovik-ice** [comment]
On my end, I suspect this file was generated when I went from T.16000 stick to a VKB Gladiator NXT. Since I got the NXT I have only made profile for the Hornet, but not my other modules.
So,I went through the control assignments of the modules I had not touched, but still DCS creates the file whenever I click OK on the controls tab.
The difference here in my case the disabled.lua does not list any devices:
`local disabled = {
	["devices"] = {
	},
	["pnp"] = false,
}
return disabled`
